packages/app/src/routers/SavedviewsRouter/components/ComponentInputs.tsx

```diff
@@ -0,0 +1,177 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *
+ * This code is for demonstration purposes and should not be considered production ready.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ColorDef } from "@bentley/imodeljs-common";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { ColorPickerButton } from "@bentley/ui-components";
+import { Slider } from "@itwin/itwinui-react";
+import React, { useState } from "react";
+
+import "./ComponentInputs.scss";
+import { generateJSONInStyles } from "./GenerateJSONStyles";
+import { reloadDisplayStyle } from "./ReloadDisplayStyle";
+
+export interface ComponentsColorPickerProps {
+  label: string;
+  path: any[];
+  pathLength: number;
+  dataType?: string;
+}
+export function ComponentsColorPicker({
+  label,
+  path,
+  pathLength,
+  dataType,
+}: ComponentsColorPickerProps) {
+  return (
+    <div className="label-with-color-picker">
+      <h3 className="text-setting">{label}</h3>
+      <ColorPickerButton
+        className="color-picker-height text-setting"
+        initialColor={ColorDef.fromString("skyblue")}
```

Comment:
Initial color should be fetched from the value, as currently everything is sky blue and gives wrong information.

Reply:
set all the colors to the default color if not declared. If it is declared, set it to that color